README.md

```diff
@@ -19,97 +20,161 @@ This WordPress project is intended to provide:
+- **Install Automated Test Suite**
 
-  ```
-  mysql -uroot
-  CREATE DATABASE chappress_dev;
-  GRANT ALL PRIVILEGES ON chappress_dev.* TO "chappress"@"localhost" IDENTIFIED BY "chappress";
-  FLUSH PRIVILEGES;
-  EXIT
-  ```
+Navigate to project folder root: `/chap-press`
 
-- Download WordPress and run the server.
+```
+composer install
+#installs depedency from composer.json
 
-  ```
-  git clone git@github.com:chapmanu/chap-press.git
-  cd chap-press/public
-  cp -v wp-config.php{-dist,}
+echo "alias codecept=./vendor/bin/codecept" >> ~/.bash_profile
+#creates an alias
 
-  # The wp-config file already contains the database information to get started.
-  # These are for local development only.
+source ~/.bash_profile
 
-  php -S localhost:8000
-  ```
+codecept --version
+```
 
-  Server should be running in the terminal displaying PHP version, port location, and document root.
+Codeception executed as `codecept` or `./vendor/bin/codecept`
 
-  Go to your browser and enter the url port:
-  ```
-  http://localhost:8000
-  ```
+- **Run the Server**
 
-  Finish the final instructions through the WordPress installation. The WordPress admin panel should be displayed and the user should have full access to developing in a local environment.
+Navigate to the `/public` directory and run `php -S localhost:8000`
+
+
+Go to your browser and enter the url port: `http://localhost:8000`
+
+Server should be running in the terminal displaying PHP version, port location, and document root.
+Finish the final instructions through the WordPress installation. The WordPress admin panel should be displayed and the user should have full access to developing in a local environment.
+
+***
 
 ## Debugging
 
-  WordPress comes with specific debug systems designed to simplify the process.
-  The following are meant only for local testing and staging installs.
+WordPress comes with specific debug systems designed to simplify the process.
+The following are meant only for local testing and staging installs.
+
+The following code is already set in the `wp-config.php` file.
+It will log all errors, notices, and warnings to a file called `debug.log` in the wp-content directory.
+It will also hide the errors so they do not interrupt page generation.
+
+<br/>
+
+| Command | Description |
+| --- | --- |
+| `define( 'WP_DEBUG', true );` | Enable WP_DEBUG mode |
+| `define( 'WP_DEBUG_LOG', true );` | Enable Debug logging to `/wp-content/debug.log` |
+| `define( 'WP_DEBUG_DISPLAY', false );` | Disable display of errors and warnings |
+| `@ini_set( 'display_errors', 0 );` | // |
+| `define( 'SCRIPT_DEBUG', true );` | Uses unminified versions of core JS and CSS files |
+
+***
+
+## Automated Testing
+
+### Run Automated Tests
+
+- **Execute in terminal**: `codecept run`
+
+This command will run all tests (acceptance,functional,unit,wpunit)
+
+[See Table](#codeception-commands) below for more specific commands
+
+![Demo](http://codeception.com/images/codecept_run.gif)
```

Comment:
Using a different port for local server is fine. I really like the way tests run now. I'm ok changing local server port. In fact, I'd prefer not to use 8000 as that is a common default and more likely to lead to conflicts.